### DISCUSSION TITLE
Can included files be executed sequentially (Not in parallel.)?

### DISCUSSION THREAD

**Discussion by yu-fuku** [stepci/stepci#122] [Q&A] [answered]
We want to separate test files for better file management
But we want to run the tests sequentially, is there a way?

```
version: "1.1"
name: Status Check
include:
  - step_1.yml
  - step_2.yml
  - step_3.yml
```

Each of the included files is executed concurrently, but we want them to be executed sequentially (steps run).

**Comment by mishushakov** [comment]
Great question. Currently, you can only run tests in parallel and steps in a sequence
Will add to my list 😉

**Comment by yu-fuku** [comment, marked as answer]
I added an option to run test sequentially.
If you feel this is a good idea, I'd be happy to merge it.

Pull request on runner side (https://github.com/stepci/runner/pull/103)
Pull request on stepci side (https://github.com/stepci/stepci/pull/123)

**Comment by sebastianwis** [comment]
Hey @yu-fuku 
Thank you once again for your contribution! 
It would be amazing to welcome you to our Discord if you haven't joined it yet: 
https://discord.com/invite/KqJJzJ3BTu
This is where we keep users up to date with updates, features and bug resolutions.
